vtl_adapter/src/vtl_state_converter.cpp

```diff
@@ -84,30 +84,56 @@ void VtlStateConverter::onCommand(const MainInputCommandArr::ConstSharedPtr msg)
   auto isNotFinalized = [](const MainInputCommand& cmd) {
     return (cmd.state != MainInputCommand::FINALIZED);
   };
+
+  std::map<std::string, OutputState> output_state_arr; 
+  
   const auto& cmd_arr = msg->commands;
   const auto is_all_commands_finalized =
     (std::count_if(cmd_arr.begin(), cmd_arr.end(), isNotFinalized) == 0);
-  const auto output_state = createState(is_all_commands_finalized);
+  auto output_state = createState(is_all_commands_finalized);
```

Comment:
output_stateを以降変更していないためconstで良いと思います。
```suggestion
  const auto output_state = createState(is_all_commands_finalized);
```